js/greatPeple.js

```diff
@@ -0,0 +1,53 @@
+class GreatPeple{
+   constructor(selectorP,selectorImg){
+   this.selectorp = selectorP;
+   this.selectorimg = selectorImg;
+    
+   this.init()
+   }
+ init(){
+  let DOM =  document.querySelector(this.selectorp);
+  if(!DOM){
+      throw 'ERROR Great peple Selektor break';
+  }
+  let HTML = '';
+  HTML += `<p>It is a long established fact that a reader 
+           will be distracted by the readable 
+           content of a page when looking at its layout.</p>`
+ DOM.innerHTML=HTML; 
+  
+  let DOM2 =  document.querySelector(this.selectorimg);
+  if(!DOM){
+      throw 'ERROR Great peple Selektor break';
+  }
+  let x=0;
+  HTML = '';
+  HTML +=`   
+            <div class="col-2 greatimg">
+               <img class="greatImg" src=${greatData[x++].img}>
+            </div> 
+            <div class="col-2">
+               <img class="greatImg" src=${greatData[x++].img}>
+            </div>
+            <div class="col-2">
+              <img class="greatImg" src=${greatData[x++].img}>
+            </div>
+            <div class="col-2">
+              <img class="greatImg" src=${greatData[x++].img}>
+            </div>
+            <div class="col-2">
+              <img class="greatImg" src=${greatData[x++].img}>
+            </div>
+            <div class="col-2">
+              <img class="greatImg" src=${greatData[x++].img}>
+            </div>`
```

Comment:
šitoj vietoj protinga panaudoti for ciklą - for sukasi tiek kartų, koks greatData.length ir sumuoja šitas eilutes:
<div class="col-2">
       <img class="greatImg" src=${greatData[i].img}>
</div>

Reply:
As visada daru pirmoi greitai a paskui eigu turiu laiko tikrinu more smart ape for as taipad galvojau